sled-agent/src/opte/illumos/port_manager.rs

```diff
@@ -297,6 +313,19 @@ impl PortManager {
             "port_name" => &port_name,
         );
 
+        // Initialize firewall rules for the new port.
+        {
```

Comment:
Is the extra scope here necessary?

Reply:
Not really, no; it was just that the `rules` aren't used anywhere else, but probably a premature optimization. Removed in adc62ccc.